lib/netsuite/actions/abstract_action.rb

```diff
@@ -0,0 +1,32 @@
+module NetSuite
+  module Actions
+    class AbstractAction
+      def request(credentials={})
```

Comment:
can you link to this PR in a comment to help explain why this pattern is being used?

Reply:
I added a link to this PR to my WIP PR where I'm implementing the monkey patch for `AbstractAction`. Is that what you were referring to? https://git.corp.stripe.com/stripe-internal/pay-server/pull/596685/files#r2504037